README.md

```diff
@@ -298,6 +239,48 @@ You can use the Attributes Kit on the server too. For instance—you can generat
   ReactDomServer.renderToString(element);
   ```
 
+## API
+
+* [`render` Method](#render-method)
+* [Class Instance](#class-instance)
+* [React Component](#react-component)
+
+#### `render` Method
+
+```JavaScript
+import AttributesKit from 'attributes-kit/dist/attributes-kit';
+
+AttributesKit.render(refractElement, element, options);
+```
+
+#### Class Instance
+
+```JavaScript
+import AttributesKit from 'attributes-kit/dist/attributes-kit';
+
+let attributes = new AttributesKit({element, options});
+attributes.render(refractElement);
+```
+
+#### React Component
+
+```JavaScript
+import ReactDom from 'react-dom';
+import {Attributes} from 'attributes-kit/dist/attributes-kit';
+
+ReactDom.render(<Attributes element={refractElement} />, element);
+```
+
+### Parameters
+
+Attributes Kit always takes the same parameters.
+
+| Parameter | Description |
+|:----------|:-----------|
+| `refractElement` | [Refract](https://github.com/refractproject/refract-spec) element to be rendered. |
+| `element` | A DOM node or a CSS selector; Attributes Kit will be rendered inside the element. |
+| `options` | An options object to customize the rendering. |
```

Comment:
@Baggz 
I think we're having some confusion between using the `render` function and the component itself because of

`<Attributes element={refractElement}/>`

and

`static render(refractElement, element, options)`

In the second case, `element` is the DOM element.